scale/ingest/models.py

```diff
@@ -242,7 +243,8 @@ def get_details(self, ingest_id, is_staff=False):
         ingest = ingest.defer('source_file__workspace__json_config')
         ingest = ingest.get(pk=ingest_id)
         ingest.admin_view = is_staff
-        ingest.strike.admin_view = is_staff
+        if ingest.strike:
```

Comment:
Does scan have the same `admin_view`? If so, we should update the scan in a similar way:

```
elif ingest.scan:
    ingest.scan.admin_view = is_staff
```

Reply:
doesn't look like scans differentiate between admin/non-admin views.